bbl/i18n/ru/BambuStudio_ru.po

```diff
@@ -5763,16 +5767,16 @@ msgid "PLA Plate"
 msgstr "PLA пластина"
 
 msgid "Bamabu Engineering Plate"
-msgstr ""
+msgstr "Инженерная пластина Bambu"
```

Comment:
The names of products are transliterated, sometimes with the translation indicated in brackets. If you translate, for example, the names rather than using transliteration, you can easily mislead the listener or reader about what you are talking about, because it loses its identity. So "Bambu Engineering Plate" after Translation can looks like "Бамбу Инжиниринг Плейт" and not "Инженерная пластина Bambu"

Reply:
Напишу на русском тут...

Для начала откройте китайскую версию po, и посмотри как в оригинале написано/переведено у них на свой язык. То есть у них это переведено на свой язык, чтобы понятно было всем китайцам (и это не сбивает их с толку), а нам не надо, да?  Это у западных языков есть привычка всё писать заглавными буквами, типа торговая марка, товар и прочие причуды. Я посматриваю как написано в первоисточнике - китайской версии, если какие-то спорные моменты возникают.

Для меня это всего лишь поясняющая надпись, которая пишется на пластине, понятно что на самой пластине она на английском языке, но это не значит что не надо переводить в интерфейсе программы.  
Грубо говоря, это также как, если на майке будет написано "Reebok Red Linen T-shirt" и его надо оставлять без перевода. Имхо это всё жирное, конечно.

Согласен, что перевести в скобках было бы конечно лучше, но тогда надпись просто не влезет в заданные рамки, да и кто-нибудь точно пнул бы за это. Перевод транслитерацией в данном случае уж точное неуместен, он никак не поможет пользователю. Приходится выбирать меньшую из двух зол. Никакого заблуждения или чего-то подобного в этом нет, а наоборот становится понятней смысл каждой пластины, пусть голову хоть включают глядя на саму пластину.

Reply:
Please use English for international communication. Not all participants know what you are talk about. Anyway I think someone of Bambu lab team can provide right understanding can we translate that brand name or not.